app/src/main/java/com/kms/cura/view/LoginActivity.java

```diff
@@ -0,0 +1,94 @@
+package com.kms.cura.view;
+
+import android.content.pm.ActivityInfo;
+import android.content.res.Configuration;
+import android.graphics.Paint;
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+import android.text.Editable;
+import android.text.TextWatcher;
+import android.widget.Button;
+import android.widget.EditText;
+
+import com.kms.cura.R;
+
+public class LoginActivity extends AppCompatActivity implements TextWatcher {
+
+    EditText email;
+    EditText password;
+    Button forgotPasswordButton,loginButton,createAccountButton;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_login);
+        initView();
+    }
+
+    /**
+     * Init view with entity object
+     */
+    public void initView(){
+        //Set forgot paswword button style
```

Comment:
Typo "paswword" for comment. Instead of comment, you could try split initView() into sub methods for better understanding. For example in this initView() can have following methods : initPasswordButton(), initCreateAccountButton(), initLoginButton(), initEmailAndPasswordText()